include/openPMD/IO/ADIOS/ADIOS2IOHandler.hpp

```diff
@@ -40,6 +39,11 @@
 #include <utility> // pair
 #include <vector>
 
+#include <nlohmann/json.hpp>
```

Comment:
- json: somehow included 2x :)
- json: should be before the includes of stdlibs
- `"openPMD/config.hpp"` should be in the first block of includes :)